resources-linux/k8s/datasource_traces_utils.bash

```diff
@@ -124,3 +124,60 @@ function build_logzio_spm_token_helm_set {
     write_task_post_run "LOG_HELM_SETS+='$helm_set'"
     write_task_post_run "HELM_SETS+='$helm_set'"
 }
+
+# Gets is service graph was selected
+# Input:
+#   ---
+# Output:
+#   IS_SERVICE_GRAPH - Tells if service graph was selected (true/false)
+function get_is_service_graph_was_selected {
+    local func_name="${FUNCNAME[0]}"
+
+    local message='Getting is Service Graph option was selected ...'
+    send_log_to_logzio "$LOG_LEVEL_DEBUG" "$message" "$LOG_STEP_TRACES" "$LOG_SCRIPT_TRACES" "$func_name" "$AGENT_ID" "$PLATFORM" "$SUB_TYPE" "$CURRENT_DATA_SOURCE"
+    write_log "$LOG_LEVEL_DEBUG" "$message"
+
+    PARAMS=("${METRICS_PARAMS[@]}")
+    get_param_value 'isServiceGraph'
+    if [[ $? -ne 0 ]]; then
+        message="traces.bash ($EXIT_CODE): $(get_task_error_message)"
+        send_log_to_logzio "$LOG_LEVEL_ERROR" "$message" "$LOG_STEP_TRACES" "$LOG_SCRIPT_TRACES" "$func_name" "$AGENT_ID" "$PLATFORM" "$SUB_TYPE" "$CURRENT_DATA_SOURCE"
+        write_task_post_run "write_error \"$message\""
+
+        return $EXIT_CODE
+    fi
+
+    local is_servicegraph=$PARAM_VALUE
+
+    if $is_servicegraph; then
+        message='Service Graph option was selected'
+    else
+        message='Service Graph option was not selected'
+    fi
+    send_log_to_logzio "$LOG_LEVEL_DEBUG" "$message" "$LOG_STEP_TRACES" "$LOG_SCRIPT_TRACES" "$func_name" "$AGENT_ID" "$PLATFORM" "$SUB_TYPE" "$CURRENT_DATA_SOURCE"
+    write_log "$LOG_LEVEL_DEBUG" "$message"
+
+    write_task_post_run "IS_SERVICE_GRAPH=$is_servicegraph"
+}
+# Builds enable service graph Helm set
+# Input:
+#   ---
+# Output:
+#   LOG_HELM_SETS - Contains all the Helm sets for logging
```

Comment:
for logging?

Reply:
Yes, internal logging. Please note it's like this across all of the scripts.